spring-cloud-stream-metrics/src/main/java/org/springframework/cloud/stream/metrics/ApplicationMetrics.java

```diff
@@ -35,19 +35,15 @@
 	private final Date createdTime;
 
 	private String name;
-
-	private int instanceIndex;
```

Comment:
We already use similar `instanceIndex` semantics in `BindingServiceProperties`. But, I don't seem to find any relevant usages of that property as well.